packages/cln_plugin/lib/src/rpcmethods/manifest/getmanifest.dart

```diff
@@ -0,0 +1,63 @@
+import 'dart:collection';
+
+import 'package:cln_plugin/src/rpcmethods/abstractrpcmethod.dart';
+import 'package:cln_plugin/src/rpcmethods/manifest/types/option.dart';
+import 'package:cln_plugin/src/rpcmethods/manifest/types/feature.dart';
+
+class ManifestMethod extends AbstractRpcMethod {
+  /// List of methods exposed
+  late List<AbstractRpcMethod> rpcMethods;
```

Comment:
```suggestion
  late HashMap<String, RpcMethod> rpcMethods;
```

Reply:
This can be also a set, but we make a little bit the implementation hard to read, I like to call it more magic, I will reflect on this